src/cache/cache.jl

```diff
@@ -176,6 +176,8 @@ function build_cache(
             unit_basis_vector_data.(CT3, sfc_local_geometry)
         ),
         ᶜh_ref,
+        T_ref,
```

Comment:
Let's use a constant T_r, as in Herrington. So this should become a ClimaParams quantity, not a cache.